.github/renovate.json

```
@@ -1,6 +1,11 @@
 {
 	"$schema": "https://docs.renovatebot.com/renovate-schema.json",
 	"automerge": true,
+	"extends": [
+		"config:recommended",
```

[Question] Based on https://docs.renovatebot.com/presets-config/#configbest-practices, it looks to me that `config:best-practices` includes `config:recommended` and a few other nice ones. Is there a reason not to go with `config:best-practices`?

```suggestion
		"config:best-practices",
```

---

Maybe also a docs issue on https://docs.renovatebot.com/presets-helpers/#helperspingithubactiondigests. "digests" is ambiguous: is it pinning to commit/hash? tag? etc.? ☹️